lib/praxis/action_definition.rb

```diff
@@ -31,7 +31,7 @@ def self.decorate_docs(&callback)
     end
 
     def initialize(name, resource_definition, **opts, &block)
-      @name = name
+      @name = name.to_sym
```

Comment:
we don't. Here's what I'll do:
I'll check in the resource definition before creating the action (and blow up if it's not a symbol).
I'll remove the sym conversion in the action initialization